contrib/test_jenkins.sh

```diff
@@ -379,7 +379,7 @@ run_coverity() {
 		cov_build="$WORKSPACE/$cov_build_id"
 		rm -rf $cov_build
 		cov-build   --dir $cov_build $MAKE all
-		cov-analyze --dir $cov_build
+		cov-analyze $COV_OPT --dir $cov_build
```

Comment:
currently we don't have any options so de facto we have empty var..
however i can add doc to test_jenkins.sh on which env vars affect the script